src/main/scala/CypherExpressable.scala

```diff
@@ -25,4 +26,16 @@ object CypherExpressible {
     def toQuery(double: Double): String =
       double.toString
   }
+
+  implicit object CypherExpressibleBoolean extends CypherExpressible[Boolean] {
+    def toQuery(boolean: Boolean): String =
+      if (boolean) "TRUE" else "FALSE"
+  }
+
+  implicit def optionExpressible[T](implicit serializer: CypherExpressible[T]): CypherExpressible[Option[T]] =
```

Comment:
looked at the play source to figure out how to make a generic typeclass builder for a container √√√